packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx

```diff
@@ -34,6 +34,7 @@ import Input from '../../component/form/Input/Input';
 import { SaveButton } from '../../component/view/savebutton';
 import { SdfSerializer } from 'ketcher-core';
 import classes from './template-lib.module.less';
+import accordionClasses from '../../../../components/Accordion/Accordion.module.less';
```

Comment:
Maybe we can have some Accordion wrapper and apply this class there rather than to import it in each file we use this Accordion?